docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,45 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work.
+
+If you ever encounter a link that helped you or that is not
+a part of the document and should be, feel free to suggest it
+by creating an issue in our `issue tracker <https://github.com/coala/coala/issues/new>`_.
+
+Git-Links
+---------
```

Comment:
Don't really know... i feel there are not enough links now for sub headings ... maybe in the future. What do you think. Still no network ....